src/schema/artist/__tests__/insights.test.ts

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable promise/always-return */
+import { runQuery } from "test/utils"
+
+declare global {
+  const sinon: any
+}
+
+describe("ArtistInsights type", () => {
+  let artist = null as any
+  let rootValue = null as any
+
+  beforeEach(() => {
+    artist = {
+      id: "foo-bar",
+      name: "Foo Bar",
+    }
+    rootValue = {
+      artistLoader: sinon.stub().returns(Promise.resolve(artist)),
```

Comment:
This doesn't need `sinon.stub()` (prefer Jest mocks if you need it).

Instead, just `artistLoader: () => Promise.resolve(artist)` should work.